syft/frameworks/torch/tensors/interpreters/replicated_shared.py

```diff
@@ -270,6 +280,130 @@ def shape(self):
     def players(self):
         return self.__get_players()
 
+    def set_garbage_collect_data(self, value):
+        shares = self.child
+
+        for _, shares in shares.items():
+            assert shares[0].is_wrapper is True
+            assert shares[1].is_wrapper is True
+
+            shares[0].gc = value
+            shares[1].gc = value
+
+    def get_garbage_collect_data(self):
+        shares = self.child
+        res = None
+
+        """ Select the first share """
+        ref_share = next(iter(shares.values()))[0]
+
+        is_wrapper = ref_share.is_wrapper
+        if is_wrapper:
+            ref_share = ref_share.child
+
+        gc_ref = ref_share.garbage_collect_data
+
+        for worker, shares in shares.items():
+            assert shares[0].is_wrapper == is_wrapper
+            assert shares[1].is_wrapper == is_wrapper
+
+            shares0_unwrap = shares[0]
+            shares1_unwrap = shares[1]
+
+            if shares[0].is_wrapper:
+                shares0_unwrap = shares[0].child
+                shares1_unwrap = shares[1].child
+
+            """ Make sure the GC value is the same for all shares """
+            assert shares0_unwrap.garbage_collect_data == gc_ref
+            assert shares1_unwrap.garbage_collect_data == gc_ref
+
+        return gc_ref
+
+    @property
+    def grad(self):
+        """
+        Gradient makes no sense for Replicated Shared Tensor, so we make it clear
+        that if someone query .grad on a Replicated Shared Tensor it doesn't error
```

Comment:
may be --> doesn't throw an error also the line "returns grad and can't be set" doesn't make much sense

Reply:
Copy pasted it from Additive Sharing Tensor. Fixed it now :D (also in AST)